### ISSUE TITLE
Incorrect examples

### ISSUE THREAD

**Issue by MaximilianYan** [GassaFM/interpr#43]
Repository has 10 code examples and unlike the other 8, "[all.pr](https://github.com/GassaFM/interpr/blob/master/examples/all.pr)" & "[simple.pr](https://github.com/GassaFM/interpr/blob/master/examples/simple.pr)" crash when trying to run them.

It's not obvious that failing is caused by examples, so newcomer might be confused when testing his system on correctness.

Also an "[arithmetic.pr](https://github.com/GassaFM/interpr/blob/master/examples/arithmetic.pr)" example does not contain ` 
   if id == 0:` line, so it is impossible to quickly check the correctness of its output.

**Comment by GassaFM** [comment]
Yeah, at this stage, perhaps the best action is to fix them so that they compile (preserving style etc. -- for example, `all.pr` shows different spacings, and that should be preserved).
